docs/admin/admin-ui/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+- installation
+- license
+---
+
+# Overview
+
+Gluu Flex Admin UI is a reactive web interface to simplify the management and configuration of your Auth Server. It is one of the key services of Gluu Flex that enables viewing and editing configuration properties, interception scripts, clients, users, metrics, etc at one place.
+
+## Installation
+
+Gluu Flex can be installed using [VM installer](../../install/vm-install/vm-requirements.md) or using [Rancher](../recipes/getting-started-rancher.md) on Cloud Native.
+
+During installation, we need to provide a Software Statement Assertion (SSA) which is used by Admin UI to register an OIDC client to access license APIs. Check the following [guide](../../install/software-statements/ssa.md) for the steps to issue SSA from the Agama Lab web interface.
+
+![image](../../assets/admin-ui/install-ssa.png)
+
+## Flex License
```

Comment:
Though not a very strict rule, try to use the brand name consistently across the documentation with the official typography. Strive to refer to the product as `Gluu Flex` instead of other variants which may get used inconsistently. Like `Flex`, `flex`, `gluu flex` etc.

Reply:
Noted and made changes.